website/addons/wiki/views.py

```diff
@@ -218,6 +218,9 @@ def project_wiki_delete(auth, wname, **kwargs):
 
     if not wiki_page:
         raise HTTPError(http.NOT_FOUND)
+    comments = getattr(node.get_wiki_page(wiki_name, 1), 'commented', [])
```

Comment:
Different wikis use different GuidStoredObjects, so if I post comments under different versions, the previous comments would not show when you update the wiki page. And so I have all comments are stored under version1 because every existing wiki has a version 1.